code/obj/item/kitchen.dm

```diff
@@ -917,6 +917,22 @@ TRAYS
 
 		. = ..()
 
+	proc/mousetrap_check(mob/user)
+		if (!ishuman(user) || is_incapacitated(user))
```

Comment:
What's this check for? Surely we want it to go off when any mob opens it, and if we've gotten here then we can assume whoever opened it was able to open it.

You should probably handle a null user though, since the calling method does. Just make sure the messages don't play if the user is null, the `triggered` procs should handle it fine.

Reply:
i didn't really think about it much, truth be told most of this i just nabbed from the storage.dm method of mousetrap trapping and repurposed it for the pizza box :p